meson.build

```diff
@@ -152,17 +147,19 @@ endif
 
 summary_msg = '---- Summary ----\n'
 summary_msg += 'Paths:\n'
-summary_msg += '  prefix          : ' + prefix + '\n'
-summary_msg += '  data dir        : ' + datadir + '\n'
-summary_msg += '  lib dir         : ' + libdir + '\n'
-summary_msg += '  locale dir      : ' + localedir + '\n'
-summary_msg += '  plugins dir     : ' + xviewer_pluginsdir + '\n'
+summary_msg += '  prefix             : ' + prefix + '\n'
+summary_msg += '  data dir           : ' + datadir + '\n'
+summary_msg += '  lib dir            : ' + libdir + '\n'
+summary_msg += '  locale dir         : ' + localedir + '\n'
+summary_msg += '  plugins dir        : ' + xviewer_pluginsdir + '\n'
 summary_msg += 'Configuration:\n'
-summary_msg += '  enable_exempi   : ' + enable_exempi.to_string() + '\n'
-summary_msg += '  enable_exif     : ' + enable_exif.to_string() + '\n'
-summary_msg += '  enable_jpeg     : ' + enable_jpeg.to_string() + '\n'
-summary_msg += '  enable_lcms     : ' + enable_lcms.to_string() + '\n'
-summary_msg += '  enable_rsvg     : ' + enable_rsvg.to_string() + '\n'
-summary_msg += '  docs            : ' + enable_docs.to_string() + '\n'
+summary_msg += '  deprecated_warnings: ' + enable_deprecation_warnings.to_string() + '\n'
+summary_msg += '  docs               : ' + enable_docs.to_string() + '\n'
+summary_msg += 'Features:\n'
+summary_msg += '  exempi             : ' + get_option('exempi').enabled().to_string() + '\n'
```

Comment:
I don't think it is particularly useful to just print the value of the feature option. What about taking this opportunity to, instead, print whether the dependency itself was found?

Reply:
How about that:
```
feat_fmt = '  @0@             : enabled: @1@, found: @2@\n'
features = {
    # display string (w/padding), option, dependency (found?)
    'exempi': [get_option('exempi'), exempi],
    'exif  ': [get_option('exif'), exif],
    'jpeg  ': [get_option('jpeg'), jpeg],
    'lcms  ': [get_option('lcms'), lcms],
    'rsvg  ': [get_option('rsvg'), rsvg]
}

foreach feat_str, feat_val : features
    summary_msg += feat_fmt.format(feat_str, feat_val[0].enabled(), feat_val[1].found())
endforeach
```
result:
```
Features:
  exempi             : enabled: true, found: true
  exif               : enabled: true, found: true
  jpeg               : enabled: true, found: true
  lcms               : enabled: true, found: true
  rsvg               : enabled: true, found: true
```
But Meson also prints whether the deps have been found before the summary itself..
```
Run-time dependency exempi-2.0 found: YES 2.5.1
Run-time dependency libexif found: YES 0.6.21
Run-time dependency libjpeg found: YES 2.0.3
Checking if "libjpeg version is 8 or greater" with dependency libjpeg compiles: YES 
Run-time dependency lcms2 found: YES 2.9
Run-time dependency librsvg-2.0 found: YES 2.48.9
```

So I don't know how helpful this output is at all..

Reply:
Some people do like a final summary, and if you look at it as a summary of components to be built then it makes a bit of sense.

My real concern here is that just printing the value of the *option* is downright misleading, for example when the build was configured using `-Dexempi=auto` and the dependency gets found, this will print "Features: exempi: false" even though the software is built with support for exempi.

The argument in favor of summaries is that they aggregate data from earlier in the log, and list all the optional dependencies in one group alongside their configured statuses.

Reply:
Ah I think I now understand what you mean. Setting options to `auto` in meson.options results in `enabled: false` in the summary print-out, because it's `'auto'` and thus not `'enabled'`. So we should instead check `dep.found()` to _print_ whether the feature is "enabled" in the build. Correct?
Also, I now think it might be better to set all features in `meson.options` to `auto`, but in `debian/rules` to `enabled`

Reply:
changed as written in my previous comment